app/models/manuscript.rb

```diff
@@ -410,18 +410,18 @@ def version_info
 
   # chapters in the manuscript not represented in mampf
   def new_chapters
-    @chapters.select { |c| c['mampf_chapter'].nil? }
-             .map { |c| [c['new_position'], c['description'], c['counter']] }
+    @chapters.select { |c| c["mampf_chapter"].nil? }
+             .map { |c| [c["new_position"], c["description"], c["counter"]] }
   end
 
   # sections in a manuscript chapter not represented in mampf
   def new_sections_in_chapter(chapter)
     sections = sections_in_chapter(chapter)
     sections.each_with_index
             .map do |s, i|
-              [s['mampf_section'], i + 1, s['description'], s['counter']]
+              [s["mampf_section"], i + 1, s["description"], s["counter"]]
             end
-            .select { |s| s.first.nil? }
+    sections.select { |s| s.first.nil? }
```

Comment:
This seems to be a breaking change: In the lines before this one, `sections` remains unchanged, so you use `sections` of l419 again, not the result of 423.

Reply:
oh, that's the one I fixed manually, and apparently not correctly ;) Here is the fix for the fix: c8addbf555